### PR TITLE
FJ Link embed 

### PR DESCRIPTION
**Pull request by plsdev89**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Type: Message. Closes #000.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-media-embed/src/mediaembedediting.js [hunk 1]
```diff
@@ -233,6 +233,52 @@ export default class MediaEmbedEditing extends Plugin {
 							'</div>'
 						);
 					}
+				},
+				{
+					name: 'FJLink',
+					url: [
+						/((?:.)+amplifyapp.com)\/(course|learning|classroom)\/([\w=?&-]+)(\/view)?/,
```
Comment (plsdev89) [inline packages/ckeditor5-media-embed/src/mediaembedediting.js]: This is for testing purpose in preview.

Comment (suvan92) [reply packages/ckeditor5-media-embed/src/mediaembedediting.js]: make sure to remove

**Review by suvan92 — APPROVED** [review]

File: packages/ckeditor5-media-embed/src/mediaembedediting.js [hunk 2]
```diff
@@ -233,6 +233,62 @@ export default class MediaEmbedEditing extends Plugin {
...
+				{
+					name: 'FJLink',
+					url: [
+						/((?:.)+amplifyapp.com)\/(course|learning|hubs|classroom|feed\/opportunities)\/([\w=?&-]+)(\/view)?/,
+						/((?:.)+flockjay.com)\/(course|learning|hubs|classroom|feed\/opportunities)\/([\w=?&-]+)(\/view)?/
+					],
+					html: match => {
+						const domain = match[ 1 ];
+						const category = match[ 2 ];
+						let contentType, id;
```
Comment (sandizzle) [inline packages/ckeditor5-media-embed/src/mediaembedediting.js]: rename id -> contentId

**Review by sandizzle — APPROVED** [review]

File: packages/ckeditor5-media-embed/src/mediaembedediting.js (hunk 2 above)
Comment (plsdev89) [reply packages/ckeditor5-media-embed/src/mediaembedediting.js]: Done

**Pull request merged**